week-6/Homework/mandatory/1-alarmclock/style.css

```diff
@@ -1,15 +1,85 @@
+body{
+  display: flex;
+  font-family: 'Rye', cursive;
+  font-size: xx-large;
+  background-color:#f06588 ;
+}
 .centre {
   position: fixed;
+  text-align: center;
   top: 50%;
   left: 50%;
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+.settime_font{
+  display: flex;
+  justify-content: space-evenly;
+}
 
 #alarmSet {
-  margin: 20px;
+  margin-bottom:5%;
+  margin-right: 5%;
 }
 
-h1 {
-  text-align: center;
+#timeRemaining{
+  margin-bottom: 15%;
+  font-family: 'Rye', cursive;
+  font-size: 200%;
+}
+#buttons{
+  margin-top: 10%;
 }
+@media(max-width:800px){
+  #timeRemaining{
+    font-size:x-large;
+    display: inline-flex;
+  }
+  #buttons{
+    display: inline-flex;
+    margin-top:5%; 
+    font-size: medium; 
+  }
+  .settime_font{
+    display: inline-flex;
+    justify-content: space-around;
+    font-size:medium;
+  }
+  div.settime_font{
+    display:inline-flexbox;
+  }
+}
+@media(min-width:800px) and (max-width:1500px){
+  #timeRemaining{
+    font-size:110%;
+    margin-bottom: 3rem;
+    display: inline-flex;
+  }
+  #buttons{
+    display: inline-flex;
+    margin-top:5%; 
+    font-size: large; 
+  }
+  .settime_font{
+    display: inline-flex;
+    justify-content: space-around;
+    font-size:large;
+  }
+  .centre{
+    display:grid;
+  }
+  div.settime_font{
+    display:inline-flexbox;
+  }
+  #set{
+    margin-right: 0.5rem;
+  }
+  #pause{
+    margin-right: 0.5rem;
+  }
+  #restart{
+    margin-right: 0.5rem;
```

Comment:
Repeating styles like this is usually a sign that you could be re-using a CSS class.

For example: 
```
.alarm-control-button {
   margin-right: 0.5rem;
}
```